### ISSUE TITLE
Can we link people to this model, or is it still too alpha?

### ISSUE THREAD

**Issue by ines** [pagesjaunes/spacy-french-models#4]
First, thanks for the great work – very excited to see the first community-trained spaCy model! 🎉

Is it okay if we refer people to this repo and share it on the spaCy channels like Twitter? Or should we wait until development is a bit further ahead?

**Comment by thomasgirault** [comment]
Hello Ines,
I am happy to contribute to SpaCy.
You can share the models bu I think we should wait a little bit.
I am actually testing a model that seems to reach better accuracy (F1=0.94, pos tagging).
I am busy this week but I will probably release it next week.

**Comment by ines** [comment]
No worries! That's exactly why I was asking – we were thinking about sharing a link to this repo on Twitter, but we thought you might not necessarily want hundreds of people coming here straight away 😉

Just let me know if you want us to promote it at any point in the future, though. I was also thinking about adding a section to the `spacy-models` README at some point that lists community models and their links/repos. This lets us keep a nice separation between "official" and "unofficial" models and still makes it easy for people to find additional models.

**Comment by thomasgirault** [comment]
Yes, it would be nice to share our models from your GitHub repository and Twitter.
I am sure it will help to improve the quality the models.
